src/amd/compiler/aco_instruction_selection.cpp

```diff
@@ -1700,6 +1700,7 @@ void visit_alu_instr(isel_context *ctx, nir_alu_instr *instr)
          bfe->getOperand(0) = Operand(base);
          bfe->getOperand(1) = Operand(offset);
          bfe->getOperand(2) = Operand(bits);
+         bfe->getDefinition(0) = Definition(dst);
```

Comment:
thx, I'm currently reworking that anyway, because I want to use the lower_bitfield_* options in NIR.